src/App.css

```
@@ -18,6 +18,13 @@ html {
   height: 100%;
 }
 
+html.lock-scroll,
+html.lock-scroll body {
+  height: calc(var(--window-inner-height) - 1px);
+  overflow: hidden;
+  box-sizing: border-box;
+}
+
```

```suggestion
/* html.lock-scroll, */
html.lock-scroll body {
  height: calc(var(--window-inner-height) - 1px);
  overflow: hidden;
  box-sizing: border-box;
  touch-action: none;
}
```
@GrasshopperBears @14KGun 이거 이렇게 하면 주소바도 사라지고, 모달 떴을때 body가 맨 위로 스크롤되지도 않는 것 같습니다..! 드디어 해결...?!?!